node_packages/bazed-svelte/src/lib/log.ts

```diff
@@ -32,7 +32,7 @@ const firebug = (level: Level, location?: string, message?: any[], ...params: an
 }
 
 export const LOGGERS: Logger[] = [firebug]
-export let LEVEL: Level = "debug"
+export const LEVEL: Level = "debug"
```

Comment:
I was thinking that this may be adjusted at runtime, don't know if it shouldn't be const, in that case.

Reply:
Also fixed